internals/build.rs

```diff
@@ -59,6 +59,9 @@ fn write_macro(mut macro_file: impl io::Write, msrv_minor: u64, minor: u64) -> i
     writeln!(macro_file, "/// Currently only the `>=` operator is supported.")?;
     writeln!(macro_file, "#[macro_export]")?;
     writeln!(macro_file, "macro_rules! rust_version {{")?;
+
+    // These tow loops are the magic; we output the clause if_yes/if_no
```

Comment:
Nit: `s/tow/two/`

Reply:
Thanks! How and when you pop up is a real mystery to me :)

Reply:
oo, I guess "... magic" in the title geeked snapped you.